services/bower/bower-version.service.js

```diff
@@ -37,10 +42,11 @@ module.exports = class BowerVersion extends BaseBowerService {
     return { label: 'bower' }
   }
 
-  async handle({ vtype, packageName }) {
+  async handle({ packageName }, queryParams) {
     const data = await this.fetch({ packageName })
+    const includePrereleases = queryParams.include_prereleases !== undefined
 
-    if (vtype === 'v') {
+    if (!includePrereleases) {
```

Comment:
Isn't more readable without this negated condition? (https://eslint.org/docs/rules/no-negated-condition)
```js
    if (includePrereleases) {
      if (data.latest_release_number) {
        return renderVersionBadge({ version: data.latest_release_number })
      }
    } else {
        if (data.latest_stable_release) {
          return renderVersionBadge({ version: data.latest_stable_release.name })
        }
    }
```